specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/ApplicationGroups.json

```diff
@@ -0,0 +1,327 @@
+        "tags": [
+          "ApplicationGroup"
+        ],
+        "operationId": "ApplicationGroup_Get",
+        "x-ms-examples": {
+          "ApplicationGroupGet": {
+            "$ref": "./examples/ApplicationGroup/ApplicationGroupGet.json"
+          }
+        },
+        "description": "Gets an ApplicationGroup for a Namespace.",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "../../../common/v2/definitions.json#/parameters/ApplicationGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace ApplicationGroup returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/ApplicationGroup"
+            }
+          },
+          "default": {
+            "description": "Eventhub error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ApplicationGroupListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationGroup"
+          },
+          "description": "Result of the List Application Groups operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains an incomplete list of Authorization Rules"
+        }
+      },
+      "description": "The response from the List Application Groups operation."
+    },
+    "ApplicationGroup": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "type": "object",
+          "x-ms-client-flatten": true,
+          "properties": {
+            "updatedAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was updated"
+            },
+            "createdAt": {
+              "readOnly": true,
+              "format": "date-time",
+              "type": "string",
+              "description": "Exact time the Application Group was created."
+            },
+            "isEnabled": {
+              "type": "boolean",
+              "description": "Determines if the status of connection from client to namespace under this Application Group is enabled or disabled."
```

Comment:
fix: `Determines if Application Group is allowed to create connection with namespace or not. Once the policy is created, connections if any for that application group will be immediately broken `.

Reply:
Shouldn't this be - Once the isEnabled is set to false, all the existing connections will be dropped and no new connections will be allowed?

Reply:
yup, this is more concise. a small change - `all the existing connections of application group..`